installation/install-by-deb.md

```diff
@@ -42,26 +49,10 @@ For Ubuntu Bionic:
 ```bash
 # td-agent 4
 curl -fsSL https://toolbelt.treasuredata.com/sh/install-ubuntu-bionic-td-agent4.sh | sh
-# td-agent 3
+# td-agent 3 (EOL)
 curl -fsSL https://toolbelt.treasuredata.com/sh/install-ubuntu-bionic-td-agent3.sh | sh
 ```
 
-For Ubuntu Xenial:
-
-```bash
-# td-agent 4
-curl -fsSL https://toolbelt.treasuredata.com/sh/install-ubuntu-xenial-td-agent4.sh | sh
```

Comment:
`xenial` still shouldn't be dropped because we announced that we'll drop it at next minor release v4.4.0, current is still v4.3.1:

* https://www.fluentd.org/blog/td-agent-v4.3.0-has-been-released
* https://www.fluentd.org/blog/td-agent-v4.3.1-has-been-released

Reply:
Oops, it should be fixed.